src/database/models/availabilities.js

```diff
@@ -0,0 +1,48 @@
+'use strict'
+module.exports = (sequelize, DataTypes) => {
+	const Availability = sequelize.define(
+		'Availability',
+		{
+			id: {
+				allowNull: false,
+				autoIncrement: true,
+				primaryKey: true,
+				type: DataTypes.INTEGER,
+			},
+			user_id: {
+				allowNull: false,
+				type: DataTypes.INTEGER,
+			},
+			event_name: { type: DataTypes.STRING, allowNull: false },
```

Comment:
@nevil-mathew how the event will be created?

Reply:
Via CURD apis